resources/environment.go

```diff
@@ -94,6 +95,57 @@ func UpdateEnvironment(db *gorm.DB, environment *Environment) (bool, error) {
 	return result.RowsAffected == 1, result.Error
 }
 
+// UpdateEnvironmentService will update an existing Environment's service in K8s
+func UpdateEnvironmentService(db *gorm.DB, client kube.APIInterface, environment *Environment, ports []ContainerPort) (bool, error) {
```

Comment:
I think we should look at "getting" the service here, and updating, instead of in the client. That way the client just receives the service to update and updates it. What do you think? @jlaswell @larkmullins